packages/datadog-instrumentations/src/mongodb.js

```diff
@@ -29,7 +29,7 @@ const collectionMethodsWithTwoFilters = [
 
 const startCh = channel('datadog:mongodb:collection:filter:start')
 
-addHook({ name: 'mongodb', versions: ['>=3.3'] }, mongodb => {
+addHook({ name: 'mongodb', versions: ['>=3.3 <5', '>=5 <6', '>=6'] }, mongodb => {
```

Comment:
I think that also works
```suggestion
addHook({ name: 'mongodb', versions: ['>=3.3 <5', '5', '>=6'] }, mongodb => {
```